velox/common/memory/MemoryUsageTracker.h

```diff
@@ -110,48 +127,64 @@ class MemoryUsageTracker
 
   /// Increments outstanding memory by 'size', which is positive for allocation
   /// and negative for free. If there is no reservation or the new allocated
-  /// amount exceeds the reservation, propagates the change upward. Sometimes
-  /// the memory pool wants to mock an update for quota accounting purposes and
-  /// different memory usage trackers can choose to accommodate this
-  /// differently.
-  virtual void update(int64_t size);
+  /// amount exceeds the reservation, propagates the change upward. If 'size' is
+  /// zero, the function does nothing.
+  void update(int64_t size);
 
   /// Returns the current memory usage.
+  ///
+  /// TODO: we will add tracker type to calculate the current bytes based on the
```

Comment:
I'm not sure I understand this comment. Would you clarify?

Reply:
Let's chat offline. Thanks!